### ISSUE TITLE
12.0.0: Items animating to the wrong place during drag

### ISSUE THREAD

**Issue by bhollis** [atlassian/react-beautiful-dnd#1579]
### Expected behavior

Items in the list stay where they were, unless disturbed by dragging (11.0.5):
![react-b-dnd-old](https://user-images.githubusercontent.com/313208/68001954-a8f20900-fc23-11e9-9709-1f5e734dff93.gif)

### Actual behavior

Items shift up by one item's-height while dragging (12.0.0):

![react-b-dnd](https://user-images.githubusercontent.com/313208/68001814-0174d680-fc23-11e9-9e86-fa73806037ac.gif)

### Steps to reproduce

Upgrade from 11.0.5 to 12.0.0. The component in question is:
https://github.com/DestinyItemManager/DIM/blob/master/src/app/settings/SortOrderEditor.tsx

### What version of `React` are you using?

16.11.0

### What version of `react-beautiful-dnd` are you running?

12.0.0

### What browser are you using?

Chrome 78.0.3904.70, macOS

### Demo

CodeSandbox appears to be somewhat broken right now :-/

**Comment by alexreardon** [comment]
Hi there!

Thanks for raising this issue. Can you please create a standalone example on `codesandbox.io` using our boilerplate: https://codesandbox.io/s/k260nyxq9v

Without a standalone example, we will not be able to action this one

Cheers!

**Comment by alexreardon** [comment]
Also, check your console and see if you are getting any warnings

**Comment by bhollis** [comment]
Unfortunately, I can't cause it to reproduce in the sandbox. It must be something about my page's styles that are causing react-beautiful-dnd to calculate sizes and positions incorrectly (though there aren't any particularly interesting styles being applied to those elements). I tried disabling all the styles for those elements and all their parents, and the bug still occurred. The only thing I've changed is to upgrade react-beautiful-dnd.

My console has no warnings.
